app/code/Magento/InventoryApi/Api/GetProductQuantityInStockInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api;
```

Comment:
>whether namespace under which I put the service is correct;

Yes, the namespace is correct. 
As we are going to expose this service as a Public API being used in client code - it should be placed in the `Magento\InventoryApi\Api`